brianlong/www.validators.app
# 178085174_support_enhanced_characters

#### What's this PR do?
- Change encoding to UTF-8 

#### How should this be manually tested?
- Run tests
- Run `rails r script/validators_get_info.rb` and check if you have no errors and if the info is updated (check https://www.pivotaltracker.com/story/show/178085174/comments/224145114)

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/178085174

#### Task completed checklist:

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
